app/src/main/java/fr/free/nrw/commons/nearby/WikiFeedback.java

```diff
@@ -0,0 +1,35 @@
+package fr.free.nrw.commons.nearby;
+
+import android.os.Bundle;
+import android.widget.TextView;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import androidx.appcompat.app.AppCompatActivity;
+import butterknife.BindView;
+import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.mwapi.MediaWikiApi;
+
+public class WikiFeedback extends AppCompatActivity {
```

Comment:
`WikidataFeedback` would be a better name I think since it is only about Wikidata.

Reply:
Also, javadoc is required for all classes and all methods. Please read https://github.com/commons-app/apps-android-commons/wiki/Code-style to know all of the project's rules :-)

Reply:
Sure Thanks for the resource :)